src/templates/apiPage.tsx

```diff
@@ -102,7 +97,7 @@ export default props => {
   useEffect(() => {
     Prism.highlightAll();
   }, []);
-
+  console.log(bodyParameters);
```

Comment:
@leeandher we should remove this console.log

Reply:
oops, thanks 👍